bedrock/settings/__init__.py

```diff
@@ -28,16 +28,13 @@
 ]
 _csp_img_src = [
     "data:",
-    "mozilla.org",
```

Comment:
👍 `'self'` covers this already

Reply:
It's basically included in `"img-src": list(set(_csp_default_src + _csp_img_src))` with `_csp_default_src` still including `csp.constants.SELF, "*.mozilla.net", "*.mozilla.org", "*.mozilla.com"` (I haven't changed that bit, the defaults still include all the hosts and are appended to all the rules. I'm only trimming the default for `report-only` after all of this gets set.)

That's perhaps another thing to RO-testdrive, restrict the default img src hosts to just self (+GA) for images, to see if all are contained internally?

(Not sure if all img src refs are always locally to bedrock, from /media root of the instance even for CMS content in all the envs, demos, staging/integration hosts etc. — e.g. *stage.gcp.moz.works definitely links images from allizom etc., will have to look how the dev extras are being added, so we don't axe the bit that appends those for non-prod use…)